python/apps/taiga/src/taiga/stories/comments/api.py

```diff
@@ -93,3 +96,55 @@ async def list_story_comments(
     )
     api_pagination.set_pagination(response=response, pagination=pagination)
     return comments
+
+
+##########################################################
+# update story comments
+##########################################################
+
+# TODO
+
+
+##########################################################
+# delete story comments
+##########################################################
+
+
+@routes.comments.delete(
+    "/projects/{project_id}/stories/{ref}/comments/{comment_id}",
+    name="project.story.comments.delete",
+    summary="Delete story comment",
+    responses=ERROR_404 | ERROR_403,
+    status_code=status.HTTP_204_NO_CONTENT,
+)
+async def delete_story_comment(
+    request: AuthRequest,
+    project_id: B64UUID,
+    ref: int,
+    comment_id: B64UUID,
+) -> None:
+    """
+    Delete a comment
+    """
+    story = await get_story_or_404(project_id=project_id, ref=ref)
+    comment = await get_story_comment_or_404(comment_id=comment_id, story=story)
+    await check_permissions(permissions=DELETE_STORY_COMMENTS, user=request.user, obj=comment)
```

Comment:
typo: permission name should be the same as the function: "DELETE_STORY_COMMENT"